src/pymortests/operators.py

```diff
@@ -292,21 +292,28 @@ def test_restricted(operator_with_arrays):
 def test_InverseOperator(operator_with_arrays):
     op, mu, U, V = operator_with_arrays
     inv = InverseOperator(op)
+    if is_windows_platform() and 'Block' in op.__class__.__name__:
+        rtol = atol = 2e-13
```

Comment:
So what's the issue here? An 'on windows we expect less accuracy' test is kind of strange. I'd rather have it failing or increase the tolerance ...